packages/ag-charts-enterprise/src/features/annotations/annotationProperties.ts

```diff
@@ -1,6 +1,14 @@
-import { type Direction, _ModuleSupport, _Util } from 'ag-charts-community';
-
-import type { ValidationContext } from './annotationTypes';
+import {
+    type AgAxisLabelFormatterParams,
```

Comment:
Generally, it'd be good if we could avoid importing options types into the properties. Even if it means duplicating some typings.

Typescript will let us know if they are incompatible when used, but it stops them being tied directly together.